### ISSUE TITLE
PDF.js not rendering when it should

### ISSUE THREAD

**Issue by ShanaLMoore** [notch8/adventist_knapsack#880]
# Summary

example:
https://adl.b2.adventistdigitallibrary.org/concern/published_works/20218025_ellen_white_and_her_work

![Image](https://github.com/user-attachments/assets/e8999c79-2401-4fbb-a999-200f6c76c02e)

# Acceptance Criteria

- [ ]

# Screenshots or Video

# Testing Instructions

To be filled out by dev

# Notes

**Comment by KatharineV** [comment]
I'm seeing some unsplit works (uploaded before the cutover) are failing to show the PDF in the PDF.js viewer, but the thumbnail is displaying in the UV. So weird!

Example 1: https://adl.b2.adventistdigitallibrary.org/concern/published_works/20122816_the_laws_of_life_november_0_1863

**Comment by KatharineV** [comment]
Dropping a comment on behavior I'm seeing. A couple weeks ago when the team ran our big OAI import post cutover, the new works showed in PDF.js. Today I'm running a big import (but small compared to that OAI feed) and I'm seeing thumbnails only. The viewer isn't rendering. The import is still going, so maybe the viewer is just slow? But that is not how it behaved before the cutover. Before recent changes, PDF.js would render within a few seconds of a file attaching to a work. Generally I only had to refresh once.

Here's the first work that came from the recent import. It was ingested over 20 hours ago, and it shows only a thumbnail. 
https://adl.b2.adventistdigitallibrary.org/concern/published_works/38016901_signs_of_the_times_april_25_1916

And the importer still in progress: 
https://adl.b2.adventistdigitallibrary.org/importers/523?locale=en

I definitely want to check this again to confirm the viewer is still missing, but for now, here's a comment to mark what i see.

**Comment by laritakr** [comment]
This seems to be a case of re-ingesting things that were previously ingested... the link you provided was ingested a year ago, by the dates on the child works. The child works will still be in fedora. I haven't checked the parent, so I don't know what happens with the reingest... I assume it's in Valkyrie now. 

Because there are child works, it tries to use the universal viewer rather than the PDFjs viewer. And the universal viewer can't handle the mixed situations well.

In short, I think we can expect to have viewer issues until we complete the migration to Valkyrie.

**Comment by KatharineV** [comment]
PDF.js is rendering as I expect now. Uploads to Staging that are awaiting UV splitting will immediately render in PDF.js. After splitting, they switch to the UV. 

Example work I uploaded today: https://adl.s2.adventistdigitallibrary.org/concern/theses/wdf_0003_031__the_contributions_of_twelve_men_who_graduated_from_oakwood_co?locale=en

PDF.js showing:

![Image](https://github.com/user-attachments/assets/7f36a42e-727c-4c9e-9a52-60df8ba9d7f9)

UV showing after split (see ticket #881 for notes on display issues--at least the UV is here! It's just not right yet):

![Image](https://github.com/user-attachments/assets/d6204eff-f6e8-466d-950f-7518534b8e3e)

**Comment by KatharineV** [comment]
All recent tests in demo.b2 and all recent uploads to the newest tenant (spectrum) are rendering PDF.js exactly as expected.
